src/include/gurt/common.h

```diff
@@ -156,18 +159,22 @@ extern "C" {
 		int _esz = (int)(size);					\
 		int _sz = (int)(size) * (cnt);				\
 		/* Compiler check to ensure type match */		\
-		typeof(newptr) optr = oldptr;				\
+		__typeof__(newptr) optr = oldptr;			\
 		D_ASSERT((void *)&(newptr) != &(oldptr));		\
 		(newptr) =  realloc(optr, (_sz));			\
 		if ((newptr) != NULL) {					\
 			if ((cnt) <= 1)					\
 				D_DEBUG(DB_MEM,				\
-					"realloc '" #newptr "': %i at %p (old '" #oldptr "':%p).\n", \
+					"realloc '" #newptr "': %i at   \
+					%p (old '" #oldptr "':%p).\n",  \
```

Comment:
(style)  Avoid unnecessary line continuations
(style)  Avoid line continuations in quoted strings